src/generate_mhq_sample/10_generate_mhq_terr_sample.Rmd

```diff
@@ -260,46 +327,44 @@ sample_1330_da <- sample_1330_da_orig %>%
            type = factor(type, levels = levels(types$type)),
            legacy_site = TRUE,
            phab = 100,
-           sac = 1)
+           sac = 1,
+           certain = TRUE) %>%
+    left_join(coord_1330_da, by = "id")
+
+sample_1330_da_sf <- st_as_sf(sample_1330_da, coords = c("x", "y"), crs = 31370)
+
+sample_1330_da_grts <- sample_1330_da_sf %>%
+    mutate(grts_ranking = grts_master[as(., "Spatial")]) %>%
+    st_join(sac_map) %>%
+    st_join(habitatmap_pol) %>%
+    st_drop_geometry() %>%
+    left_join(coord_1330_da, by = "id") %>%
+    mutate(polygon_id = as.character(polygon_id)) %>%
+    select(id,  polygon_id, grts_ranking, phab, x, y, main_type, type, certain, description_orig, legacy_site, sac)
 
 #write.csv2(select(sample_1330_da, id), "../../output/meetpunten_1330_da.csv", row.names = FALSE)
 
 check_unique_id <- nrow(sample_1330_da) == n_distinct(sample_1330_da$id)
 
 ```
 
-# Combine files
+### Combine files
 
 ```{r}
 samples <- bind_rows(
     sample_6510,
     sample_grassland_marshes,
+    sample_dunes,
     sample_heath,
     sample_forest_new,
     sample_vbi2_type,
-    sample_1330_da
-)
-```
-
-## Read admin data
-
-```{r}
-
-admin_anb_orig <- read.csv2("../../data/admin/anb/PlotsMetDataAFGEWERKT.csv")
+    sample_91E0_sf,
+    sample_1330_da_grts
+) %>%
+    mutate(location_id = ifelse(is.na(location_id), as.character(grts_ranking), location_id)) %>%
+    arrange(type, grts_ranking) %>%
+    select(sample_id = id, location_id, everything())
 
-admin_anb <- admin_anb_orig %>%
-    mutate(circleplot_measured = ifelse(!is.na(Sitedescr_6510), Sitedescr_6510,
-                                        ifelse(!is.na(Sitedescr_heide), Sitedescr_heide, NA))) %>%
-    select(id = SingleID, obs_circleplot = Hab_Standdesc, obs_vegplot = hab_VegPQ, vegplot_measured = VEGPQ, circleplot_measured, notes = opm_navplot) %>%
-    mutate(obs_vegplot = ifelse(obs_vegplot != "geen habitat (akker, houtkant, tuin,...)", 
-                                str_replace(obs_vegplot, " ", "_"),
-                                obs_vegplot),
-           obs_circleplot = ifelse(obs_circleplot != "geen habitat (akker, houtkant, tuin,...)", 
-                                str_replace(obs_circleplot, " ", "_"),
-                                obs_circleplot))
+write_vc(samples, file = "mhq_terr_samples", root = "../../n2khab_mhq_data", sorting = c("type", "sample_id"), strict = FALSE)
```

Comment:
Some output, added simply as a reference here

```r
> glimpse(mhq_terr_samples %>% slice(seq(1,6500,500)))
Rows: 13
Columns: 14
$ sample_id        <chr> "ZSCPQ007", "11578161_2120", "11206069_2160", "12526769_2180", "95158_2310", "28232…
$ polygon_id       <chr> "562222_v2018", "BWK335", "BWK1206", "PINK90", "41039", "44562", "47029", "32636", …
$ grts_ranking     <dbl> 61453757, 11578161, 11206069, 12526769, 95158, 282322, 6951250, 11497522, 439878, 2…
$ x                <dbl> 144113.5, 36477.6, 59261.6, 24605.6, 216893.6, 224253.6, 247613.6, 219741.6, 169501…
$ y                <dbl> 222439.9, 206702.1, 221198.1, 198446.1, 199470.1, 193902.1, 188014.1, 172014.1, 168…
$ main_type        <fct> 1330, 2120, 2160, 2180, 2310, 4010, 6120, 6410, 6430, 6510, 7140, 9120, 9160
$ type             <fct> 1330_da, 2120, 2160, 2180, 2310, 4010, 6120, 6410_mo, 6430_hf, 6510_hu, 7140_meso, …
$ phab             <dbl> 100, 15, 50, 100, 100, 100, 100, 20, 70, 100, 100, 100, 100
$ sac              <dbl> 1, 1, 1, 1, 1, 1, 1, 0, 1, 1, 1, 0, 1
$ year_planned     <dbl> NA, 8, 9, 9, 3, 6, 4, 7, 8, 11, 7, NA, NA
$ description_orig <chr> "100% 1130; 100% rbbmr", "80% 2130_hd; 15% 2120; 5% 2160", "50% 2160; 25% 2130_hd; …
$ legacy_site      <lgl> TRUE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, TRUE, T…
$ certain          <lgl> TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE
$ location_id      <chr> "61453757", "11578161", "11206069", "12526769", "95158", "282322", "6951250", "1149…
```

Reply:
Considering the variable `legacy_site`, it is not yet clear which design is involved: judgment sampling, SRS etc., and whether the considered target population covered the whole of Flanders. Those attributes may be useful for your inferences later.